magic/fetcher.py

```diff
@@ -3,14 +3,70 @@
 import os
 from collections import OrderedDict
 from urllib import parse
+from functools import wraps
+import time as py_time
 
 import pkg_resources
 from github import Github
 
 import magic.fetcher_internal as internal
 from magic.fetcher_internal import FetchException
+from magic import oracle #for card-by-name for scryfall
```

Comment:
Whoops. Probably smarter to do the cardname-to-card lookup in the main command then, or what do you suggest?